src/content/en/fundamentals/performance/resource-prioritization.md

```diff
@@ -0,0 +1,351 @@
+    <img src="images/res-prio-timeout.png"
+    alt="An example of a preload timeout error in Chrome Developer Tools">
+  </div>
+</figure>
+
+### Use-case: Fonts
+
+Fonts are a great example of late-discovered resources that must be fetched,
+often sitting at the bottom of one of several CSS files loaded by a page.
+
+In order to reduce the amount of time the user has to wait for the text content
+of your site, as well as avoid jarring flashes between system fonts and your
+preferred ones, you can use `<link rel="preload">` in your HTML to let the
+browser know immediately that a font will be needed.
+
+    <link rel="preload" as="font" crossorigin="crossorigin" type="font/woff2" href="myfont.woff2">
+
+Note that the use of `crossorigin` here is important; without this attribute,
+the preloaded font will be ignored by the browser, and a new fetch will take
+place. This is because fonts are expected to be fetched anonymously by the
+browser, and the preload request is only made anonymous by using the
+`crossorigin` attribute.
+
+Caution: If you’re using a CDN, such as Google Fonts, be sure that the font files
+you’re preloading match the ones in the CSS, which can be tricky due to unicode
+ranges, weights, and font variants. Fonts can also be regularly updated, and if
+you’re preloading an old version while using the CSS for a newer one, you may
+end up downloading two versions of the same font and wasting your users’
+bandwidth. Consider using `<link rel="preconnect">` instead for easier
+maintenance.
+
+### Use-case: Critical Path CSS and JavaScript
+
+When talking about page performance, one useful concept is that of the
+“critical path”, most often applied to CSS. This refers to the resources that
+need to be loaded before your initial render, and are thus critical to getting
+the first pixels to the user’s screen.
+
+Previously, the recommendation was to inline this content into your HTML.
+However, in a multi-page, server-side rendered scenario, this quickly grows into
+a lot of wasted bytes. It also makes versioning harder, as any change in the
+critical code invalidates any page that has it inlined.
+
+`<link rel="preload">`, on the other hand, allows you to keep the benefits of
+individual file versioning and caching, while giving you mechanism to request
+the resource as soon as possible.
+
+    <link rel="preload" as="script" href="super-important.js">
+    <link rel="preload" as="style" href="critical.css">
+
+There’s a downside compared to inlining, however: with `<link rel="preload">`
+you’re still subject to an extra roundtrip. This extra roundtrip comes from the
+fact that the browser first has to fetch the HTML, and only then does it find
+out about the next resources.
+
+One way around the extra roundtrip is to use
+[HTTP/2](/web/fundamentals/performance/http2/#server_push)
+push instead, where you preemptively attach the critical assets to the same
+connection through which you’re sending the HTML. This way, you guarantee that
```

Comment:
How about 'This guarantees that there's no downtime...'

Reply:
Done.